# cd /data/openpilot && curl https://raw.githubusercontent.com/eFiniLan/openpilot-patches/master/0.5.10/redirect_crash_log_to_me.diff | git apply -v
#
# This patch will send crash log to me (eFiniLan), allow me to diagnose any porting issue.
#

 selfdrive/crash.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/selfdrive/crash.py b/selfdrive/crash.py
index 94bdfd12..640525d9
--- a/selfdrive/crash.py
+++ b/selfdrive/crash.py
@@ -18,7 +18,7 @@ if os.getenv("NOLOG") or os.getenv("NOCRASH"):
 else:
   from raven import Client
   from raven.transport.http import HTTPTransport
-  client = Client('https://1994756b5e6f41cf939a4c65de45f4f2:cefebaf3a8aa40d182609785f7189bd7@app.getsentry.com/77924',
+  client = Client('https://fa39b8804ae94ea6bbb22279d68b3dc7@sentry.io/1428745',
                   install_sys_hook=False, transport=HTTPTransport, release=version, tags={'dirty': dirty})
 
   def capture_exception(*args, **kwargs):
